tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Input/ObjcInterop/ObjcInteropTypeReader.swift

```diff
@@ -0,0 +1,197 @@
+/*
+* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+* This product includes software developed at Datadog (https://www.datadoghq.com/).
+* Copyright 2019-2020 Datadog, Inc.
+*/
+
+import Foundation
+
+/// Reads `ObjcInteropType` definitions from `SwiftType` definitions.
```

Comment:
I was confused by this comment initially.
I guess I get the overall pattern, but I would understand the role of `ObjcInteropTypeReader` more as a mapper than simply a reader (one half of it).
More specifically a mapper from `SwiftType`s to `ObjcInteropType`s, with one "step" for reading `SwiftType` definitions and and one "step" for producing `ObjcInteropType` definitions.

Reply:
Indeed, this could be better expressed as `SwiftToObjcInteropTransformer`. Because we're having also `SwiftTypeReader`, which should become `JSONSchemaToSwiftTypeTransformer`, let me do this in the next PR, to keep this one small for future references 🙏.

Reply:
Done in https://github.com/DataDog/dd-sdk-ios/pull/386